packages/playwright-core/src/web/traceViewer/traceModel.ts

```diff
@@ -213,3 +213,7 @@ export class PersistentSnapshotStorage extends BaseSnapshotStorage {
 function isTracing(metadata: CallMetadata): boolean {
   return metadata.method.startsWith('tracing');
 }
+
+function isChatty(metadata: CallMetadata): boolean {
+  return metadata.type === 'Route' && metadata.method === 'continue';
```

Comment:
What about `abort` or `fulfill`? Also, `Request.response()` and `Response.body()` come to mind.